requirements.txt

```diff
@@ -1,4 +1,18 @@
+atomicwrites==1.3.0
+attrs==18.2.0
+colorama==0.4.1
 dateparser==0.7.0
 flake8==3.6.0

```

Comment:
Requirements like flake8 or pytest should be in a different file as someone doesn't need those to run the project. Usually there is like a requirements_dev or requirements_test file only with the dependencies required.

Reply:
Excellent! Thanks for the tip, I will made the changes as suggested.